src/pgrn-query-expand.c

```diff
@@ -471,6 +472,52 @@ PGrnFindTermAttributeNumber(PGrnQueryExpandData *data,
 	return;
 }
 
+static const char*
+PGrnConvertRelationKind(char relkind)
```

Comment:
It's not a good name. It doesn't say anything about convert to.
How about `PGrnRelkindToString()` like `PGrnJSONBValueTypeToString()`?